src/server/report/result.js

```diff
@@ -162,25 +153,32 @@ function renderResult(resultData, options) {
 }
 
 export function renderBrowserResults(browser, tests, options) {
-	const results = tests.map(t => {
+	const results = tests.reduce((acc, t) => {
+
 		const resultData = t.results.find(r => r.name === browser.name);
-		const duration = resultData.duration;
-		const durationClass = {
-			'error': duration >= 1000,
-			'result-duration': true,
-			'pass': duration < 500,
-			'warning': duration >= 500 && duration < 1000
-		};
-		return html`
+		if (resultData.passed && options.filterStatus === FILTER_STATUS.FAILED ||
+			!resultData.passed && options.filterStatus === FILTER_STATUS.PASSED) {
+			return acc;
+		}
+
+		let status = STATUS_TYPE.NORMAL;
+		if (resultData.duration >= 1000) {
```

Comment:
We can do this later if we want to at all, but can we hook this value up to the `slow` mocha config?

Reply:
I did have to add it to testInfo yeah. It would be cool if `@web/test-runner` exposed it to the reporters, but it does not (from what I can tell).